src/main/java/org/mockito/internal/junit/JUnitRule.java

```diff
@@ -14,25 +13,74 @@
 public class JUnitRule implements MockitoRule {
 	
 	private final MockitoLogger logger;
+    private final boolean silent;
 
-	public JUnitRule(MockitoLogger logger) {
+    public JUnitRule(MockitoLogger logger, boolean silent) {
```

Comment:
At least the parameter `silent` should be documented. It is not clear at this point what effect the argument has.